validator/distro_validator.go

```diff
@@ -547,6 +547,20 @@ func ensureHasValidVirtualWorkstationSettings(ctx context.Context, d *distro.Dis
 			Level:   Error,
 		})
 	}
+	linuxDistros := []string{
+		evergreen.ArchLinux386,
+		evergreen.ArchLinuxPpc64le,
+		evergreen.ArchLinuxS390x,
+		evergreen.ArchLinuxArm64,
+		evergreen.ArchLinuxAmd64,
+	}
+
+	if !utility.StringSliceContains(linuxDistros, d.Arch) {
+		errs = append(errs, ValidationError{
+			Message: "workstation distros must use Linux architecture",
```

Comment:
nit: The phrasing is a little weird because Linux isn't an architecture. The arch enum represents a combination of OS + CPU architecture, so phrasing this as "workstation distros must use Linux with a supported CPU architecture" (or something like that) would sound less ambiguous.